x-pack/platform/packages/shared/kbn-inference-endpoint-ui-common/src/hooks/use_providers.ts

```diff
@@ -14,6 +14,31 @@ import * as i18n from '../translations';
 
 const getProviders = (http: HttpSetup): InferenceProvider[] => {
   return [
+    {
+      service: 'elastic',
+      name: 'Elastic',
+      task_types: ['sparse_embedding', 'completion'],
+      configurations: {
+        'rate_limit.requests_per_minute': {
```

Comment:
I'm not really familiar with the Kibana side, but does this mean, that this is exposed (and configurable) to the user?

Reply:
Yes, it is exposed and configurable by the user. All the settings in the configurations are visible and configurable in Kibana UI